api/libs/planning-unit-geometry/src/projects-pu.geo.entity.ts

```diff
@@ -40,4 +43,15 @@ export class ProjectsPuEntity {
     enumName: 'planning_unit_grid_shape',
   })
   geomType!: PlanningUnitGridShape;
+
+  // @OneToMany(
+  //   () => CostSurfacePuDataGeoEntity,
+  //   (costSurfacePuData: CostSurfacePuDataGeoEntity) =>
+  //     costSurfacePuData.projectsPu,
+  //   { onDelete: 'CASCADE' },
+  // )
+  // costSurfacePuData!: CostSurfacePuDataGeoEntity[];
+  /**
+   * @todo: solve this doubt. move all geoprocessing cost surface stuff to libs? only the relevant entity?
```

Comment:
we didn't discuss this today - please let me know if you need more info on this, or even just a functional-design rubber ducky

Reply:
Yep, I totally forgot to bring this up. Let's **extremely** quickly mention it in today's call.